cafebabel/articles/models.py

```diff
@@ -12,6 +12,12 @@
 from .tags.models import Tag
 
 
+class ArticleArchive(db.EmbeddedDocument):
+    pk = db.IntField()  # In use for migrations (references in related).
+    url = db.StringField()  # In use for redirections.
+    relateds = db.ListField()
```

Comment:
I see you had put this in the `Article` object in the first place before moving it here.
I shared the opinion of the `Article` property as we will need it the same way for all articles.
What made you change for this?

Reply:
See my other comment, here we just store old ids for later use.